debugcc.c

```diff
@@ -175,7 +179,7 @@ static unsigned long measure_default(const struct measure_clk *clk)
 		writel(xo_div4 | 1, gcc->base + gcc->xo_div4_reg);
 
 	raw_count_short = measure_ticks(gcc, 0x1000);
-	raw_count_full = measure_ticks(gcc, 0x10000);
+	raw_count_full = measure_ticks(gcc, sample_ticks);
```

Comment:
I see that you lower the sample_ticks for the CPU clocks. What would be the issue with the current value? Does it result in integer overflow somewhere?

Reply:
This is how it's done downstream. From what I notice using 10000 result in the value getting divided by 4... And the value is already * 8 ... Maybe this will result in  less precision? Will check

Reply:
Tested with 0x10000 and produce the same correct values so I dropped the patch.

Reply:
Thank you!